### PR TITLE
Add copy to clipboard functionality in messages

### PR DESCRIPTION
**Pull request by eddierayo-156**
Added the copy to the clipboard feature. Right next to the text box, there is a clip board button that once pressed, will copy the contents of the box. It will then pop up a message saying "copied!' right next to it. This message will disappear by itself so the user does not need to click anything to dismiss the message.

### CODE REVIEW

File: src/Chat.css [hunk 1]
```diff
@@ -96,3 +103,28 @@
...
+  min-height: 30px;
+}
+#custom-tooltip {
+  display: none;
+  margin-left: 8px;
+  background-color: transparent;
+  border-radius: 3px;
+  border-color: #ffffff;
+  color: #000000;
+}
```
Comment (soumikbaithalu) [inline src/Chat.css:130]: Add the extra line to  EOF

**Review by soumikbaithalu — CHANGES_REQUESTED** [review]
image
Fix the size of the sidebar chat.

**Comment by soumik2012** [comment]
#16

**Comment by soumikbaithalu** [comment]
@eddierayo-156 before adding the PR Check in the localhost if it's right or no. Also, I have seen that you push directly into the master branch please `checkout` from the `master` branch then add your function because I can't review your PR in the local machine. But your work is good by the way.

**Comment by soumikbaithalu** [comment]
Closing the issue due to no response thank you for the PR you can look into different issue

**Pull request closed without merging**